image/cli/mascli/functions/mirror_to_registry

```diff
@@ -21,6 +21,12 @@ Source Registry Entitlements (Required based on what content you choose to mirro
       --ibm-entitlement ${COLOR_YELLOW}IBM_ENTITLEMENT_KEY${TEXT_RESET}       IBM Entitlement Key
       --redhat-username ${COLOR_YELLOW}REDHAT_CONNECT_USERNAME${TEXT_RESET}   Red Hat Connect Username
       --redhat-password ${COLOR_YELLOW}REDHAT_CONNECT_PASSWORD${TEXT_RESET}   Red Hat Connect Password
+      --artifactory-username ${COLOR_YELLOW}ARTIFACTORY_USERNAME${TEXT_RESET} Artifactory Username
+      --artifactory-apikey ${COLOR_YELLOW}ARTIFACTORY_APIKEY${TEXT_RESET}     Artifactory API Key
+
+Development Mode (Optional):
+  --dev-mode ${COLOR_YELLOW}DEV_MODE_AIRGAP${TEXT_RESET}                      Mirror Pre-release or Development Builds
```

Comment:
We already have a `DEV_MODE` flag supported by install, let's re-use that rather than introduce `DEV_MODE_XXX` per command.

If you are in `DEV_MODE` you are likely to want to be in dev mode for everything, and if you don't want to be then you can set it per command instead of exporting it.